{"name": "Testing login with invalid credentials", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\Eldor\\\\PycharmProjects\\\\appetize/stage_config.json'", "trace": "self = <tests.test_login.TestLogin object at 0x000001B72B2B02B0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"88dc5af382228acbd1b9151887d5a75d\")>\nenvironment = <ENV.STAGE: (2, 'STAGE', 'stage_config')>, username = 'tomsmith'\npassword = 'wrong_pass', key = 'invalid_password_txt'\n\n    @allure.feature('Login Page')\n    @allure.title(\"Testing login with invalid credentials\")\n    @allure.testcase(\"https://accunture.testrail.io/index.php?/cases/view/9999\", 'Test Case Link')\n    @pytest.mark.C9999  # TestRail ID\n    @pytest.mark.smoke\n    @pytest.mark.parametrize(\"environment\", [ENV.PROD, ENV.STAGE])\n    @pytest.mark.parametrize(\"username, password, key\", [\n        (\"wrong_username\", \"SuperSecretPassword!\", \"invalid_username_txt\"),\n        (\"tomsmith\", \"wrong_pass\", \"invalid_password_txt\"),\n        (\"wrong_username\", \"wrong_pass\", \"invalid_username_txt\")\n    \n    ])\n    def test_login_invalid_creds(self, driver, environment, username, password, key):\n>       login_page = LoginPage(env=environment)\n\ntest_login.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\services\\herokuapp\\pages\\login_page\\__init__.py:18: in __init__\n    self.config = CommonHelper.json_result_reader(file_name=env.config)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nfile_name = 'stage_config'\n\n    @staticmethod\n    def json_result_reader(file_name) -> Dict:\n        \"\"\"\n        Open json by provided name  and return\n        :param file_name: Name of Config file without extension\n        :return: dict\n        \"\"\"\n        import json\n        path = str(CommonHelper.get_project_root())\n>       with open(path + '/' + file_name + '.json') as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\Eldor\\\\PycharmProjects\\\\appetize/stage_config.json'\n\n..\\utility\\helpers.py:119: FileNotFoundError"}, "steps": [{"name": "Add failed screenshot", "status": "passed", "attachments": [{"name": "Failed screenshot", "source": "1f584005-7be5-4b69-aa3b-a3dd51c11688-attachment.png", "type": "image/png"}], "start": 1637829835718, "stop": 1637829836238}], "attachments": [{"name": "log", "source": "3ad24d4d-2619-4582-a70f-16284f6f407c-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "9df45aee-d364-47d2-94de-39fa6217294d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'tomsmith'"}, {"name": "password", "value": "'wrong_pass'"}, {"name": "key", "value": "'invalid_password_txt'"}, {"name": "environment", "value": "<ENV.STAGE: (2, 'STAGE', 'stage_config')>"}], "start": 1637829835666, "stop": 1637829835668, "uuid": "bdb7ccf0-4150-48cb-8de4-4a6251c32228", "historyId": "1afab8c569aa6b4bedfd5e956a0163d1", "testCaseId": "d3de27ef8e58de518c7ec3f299a7ed3d", "fullName": "tests.test_login.TestLogin#test_login_invalid_creds", "labels": [{"name": "feature", "value": "Login Page"}, {"name": "tag", "value": "smoke"}, {"name": "tag", "value": "C9999"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-LKUFHBJQ"}, {"name": "thread", "value": "47252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "links": [{"type": "test_case", "url": "https://accunture.testrail.io/index.php?/cases/view/9999", "name": "Test Case Link"}]}